src/Symfony/Component/Cache/LockRegistry.php

```diff
@@ -123,9 +122,9 @@ public static function compute(callable $callback, ItemInterface $item, bool &$s
                 }
                 // if we failed the race, retry locking in blocking mode to wait for the winner
                 $logger?->info('Item "{key}" is locked, waiting for it to be released', ['key' => $item->getKey()]);
-                flock($lock, \LOCK_SH);
+                (bool) flock($lock, \LOCK_SH);
```

Comment:
I'd argue this here is technically a bug, if flock fails we probably want to throw or retry instead of just assuming the race-winner is done writing?

Reply:
if the lock fails, we'll just recompute the value so that's fine ignoring to me